### PR TITLE
 Add Log statement in exec_util.go for ClientSet Creation

### LINKED ISSUE #102295: add ExecWithOptions to exec_util.go for ClientSet Creation
We should fine grained  debug statements to ExecWithOptions in the framework exec_util.go class , so that in tests where theres alot of execs where hangs can happen i.e. on windows, we can see where tests are getting clogged up.  Mostly i want this for network_policy.go, but i suppose its a generic improvement

```
	Logf("ExecWithOptions: Clientset creation ") 
```

### PR DESCRIPTION
**Pull request by mcbenjemaa**
#### What type of PR is this?
/kind flake

#### What this PR does / why we need it:
Add some Log statement to enhance debug experience for in the framework exec_util.go
#### Which issue(s) this PR fixes:

Fixes #102295

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: test/e2e/framework/exec_util.go [hunk 1]
```diff
@@ -58,6 +58,7 @@ func (f *Framework) ExecWithOptions(options ExecOptions) (string, string, error)
 
 	const tty = false
 
+	Logf("ExecWithOptions: Clientset creation ")
```
Comment (oomichi) [inline test/e2e/framework/exec_util.go:61]: nit: Why do we need to add a space at the end of the message after `creation`?

**Review by oomichi — COMMENTED** [review]
/lgtm

File: test/e2e/framework/exec_util.go (hunk 1 above)
Comment (mcbenjemaa) [reply test/e2e/framework/exec_util.go]: /remove-extra-space

**Comment by mcbenjemaa** [comment]
Please approve to merge?

**Pull request merged**